.github/workflows/conformance.yml

```diff
@@ -62,16 +65,19 @@ jobs:
           sudo mv kind /usr/local/bin
       - name: Create K8s cluster
         run: |
+          # The command also loads local antrea/antrea-ubuntu:latest into Nodes if it exists.
           ./ci/kind/kind-setup.sh create kind \
             --k8s-version "${{ inputs.k8s-version }}"
       - name: Install Antrea
         run: |
           if [ ${{ steps.check-release.outputs.released }} == 'true' ]; then
             helm repo add antrea https://charts.antrea.io
             helm repo update
-            helm install --namespace kube-system antrea antrea/antrea --version "${{ inputs.antrea-version }}"
+            helm install --namespace kube-system antrea antrea/antrea --version "${{ inputs.antrea-version }}" \
+              --set "${{ inputs.antrea-values }}"
```

Comment:
Question: can this work properly with empty value?

Reply:
yes, it's tested